tests/phpunit/CRM/Core/BAO/FinancialTrxnTest.php

```diff
@@ -90,23 +90,18 @@ public function testGetTotalPayments() {
       'thankyou_date' => '20080522',
     );
 
-    $contribution = CRM_Contribute_BAO_Contribution::create($params);
+    $contribution = $this->callAPISuccess('Contribution', 'create', $params);
+    $contribution = $contribution['values'][$contribution['id']];
 
-    $this->assertEquals($params['trxn_id'], $contribution->trxn_id);
```

Comment:
@eileenmcnaughton are we loosing anything by removing these asserts?

Reply:
I don't think so -creating contributions is tested elsewhere & these are tangental to what is being tested